app/overrides/decorate_admin_products_form.rb

```diff
@@ -1,6 +1,7 @@
 Deface::Override.new(
   virtual_path: "spree/admin/products/_form",
   name: "multi_domain_admin_product_form_meta",
+   :original => '2724f51d519589f73401abeae8f8e05d9dae99c4',
```

Comment:
@blset Just curious how can I generate this `original` SHA1 value, from which string?